tests/Unit/inc/Engine/Optimization/Minify/CSS/Combine/optimize.php

```diff
@@ -60,8 +60,15 @@ public function testShouldCombineCSS( $original, $combined, $cdn_host, $cdn_url,
 			} );
 
 		$this->assertSame(
-			$combined,
+			$expected['html'],
 			$this->combine->optimize( $original )
 		);
+
+		foreach ( $expected['files'] as $file ) {
+			$this->assertTrue( $this->filesystem->exists( $file ) );
+		}
+
+		$this->assertTrue( $this->filesystem->exists( 'wordpress/wp-content/cache/min/1/0bca286481748a69cdbe9a6695015ec9.css' ) );
```

Comment:
These 2 assertions should be removed, right?

Reply:
You're right I forgot to delete them, just pushed the commit for it